src/schema/schema-generator.ts

```diff
@@ -498,6 +502,7 @@ export abstract class SchemaGenerator {
         astNode: getFieldDefinitionNode(handler.schemaName, type, handler.directives),
         extensions: {
           complexity: handler.complexity,
```

Comment:
It seems correct to me that `extensions.complexity` would be overridden by regular extensions, seeing as if I understood it correctly the plan is to refactor `complexity` and other decorators to use `extensions` at some point.

Bottom line: if `@Extensions` is a lower level decorator, it should take precedence over others if users would like to implement custom stuff

Reply:
Yes, the complexity decorator can be replaced by the extensions. Preventing colliding properties names is the users responsibility.